lib/ansible/runner.rb

```diff
@@ -217,14 +223,19 @@ def async?(ansible_runner_method)
         ansible_runner_method == "start"
       end
 
-      def runner_params(base_dir, ansible_runner_method, playbook_or_role_args)
+      def runner_params(base_dir, ansible_runner_method, playbook_or_role_args, verbosity)
         runner_args = playbook_or_role_args.dup
 
         runner_args.delete(:roles_path) if runner_args[:roles_path].nil?
 
         runner_args[:role_skip_facts] = nil if runner_args.delete(:role_skip_facts)
         runner_args[:ident] = "result"
 
+        if verbosity > 0
+          v_flag = "-#{"v" * verbosity}"
+          runner_args[nil] = v_flag
```

Comment:
I am sure this works with `AwesomeSpawn`, I just always thought that you did "no-arg flags" (`-vvv`) as:

```ruby
irb> AwesomeSpawn "foo" :arg1 => "val", :flag1 => nil
#=> foo --arg1=val --flag1
```

Why I ask is because it seems like in this form you could only have 1 flag, since the `nil` key only has a single value.  Correct?

Reply:
Yeah, I want a better way to do this, but wanted to get it working for the first pass.

The issue is that `:vvv => nil` produces `--vvv` rather than `-vvv`

Reply:
That is fair.  You might be able to do it with a `-v -v -v` in some way that works with the `AwesomeSpawn` interface better, but I don't think it is worth holding up this PR to try and figure that out at the moment.

Reply:
params doesn't have to be a Hash.  It can also be a regular Array, so you could do

```ruby
[:role_skip_facts, {:ident => result}, "-vvv"]
```

The nil value stuff is just convenience for when a key doesn't have a value and you still want to put it in a Hash, but it's not required.

Reply:
Alternately you can do {"-vvv" => nil} in the Hash.